overlord/devicestate/firstboot20_test.go

```diff
@@ -549,6 +559,88 @@ func (s *firstBoot20Suite) TestLoadDeviceSeedCore20(c *C) {
 	c.Assert(err, ErrorMatches, `internal error: requested inconsistent device seed: 20210201 \(was 20191018\)`)
 }
 
+func (s *firstBoot20Suite) setupSeed20DangerousGrade(c *C, withAutoImportAssertion bool) string {
```

Comment:
setup is a bit of strange verb here? testProcessAutoImportAssertions... instead

Reply:
Renamed